src/compiler/watch.ts

```diff
@@ -243,13 +243,23 @@ namespace ts {
         };
     }
 
-    export const enum WatchType {
-        ConfigFile = "Config file",
-        SourceFile = "Source file",
-        MissingFile = "Missing file",
-        WildcardDirectory = "Wild card directory",
-        FailedLookupLocations = "Failed Lookup Locations",
-        TypeRoots = "Type roots"
+    export type WatchType = WatchTypeRegistry[keyof WatchTypeRegistry];

```

Comment:
friendship ended etc etc

Reply:
Yah, string enums don't merge across files, while you can make an interface that does (via module augmentation). Downside is you have to use _assignments_ to actually create the value side. Terrible, I know.